src/ui/atoms/thumbnail/Thumbnail.tsx

```diff
@@ -1,30 +1,20 @@
-import React from "react";
 import classNames from "classnames";
 import "./Thumbnail.css";
 
-import { Text } from "../text/Text";
-
-interface IThumbnail {
+interface ThumbnailProps {
   className?: string;
-  name: string;
-  bgColor: string;
-  imageSrc?: string | null;
-  size?: "large" | "normal";
+  bgColor?: string;
+  size?: "md" | "sm";
+  children: React.ReactNode;
```

Comment:
When you're using React types, can you import using an import statement?

```js
import { ReactNode } from "react";
```

I know it's more to write, but I'm not super fond of relying on ambient types from non-builtin classes.